schemes/Material-Theme-Lighter.tmTheme

```diff
@@ -55,7 +55,7 @@
 				<key>fontStyle</key>
 				<string>italic</string>
 				<key>foreground</key>
-				<string>#A3ACAE</string>
+				<string>#BDBDBD</string>
```

Comment:
You have to change this file
https://github.com/equinusocio/material-theme/blob/develop/sources/settings/specific/Material-Theme-Lighter.json

and then rebuild the theme, not the files inside `schemes/`.

We also use a blueish gray color from material design **Blue Grey** palette, i think one of these will be good:
- #CFD8DC
- #B0BEC5
- #90A4AE